client/my-sites/plugins/marketplace/marketplace-domain-upsell/styles.scss

```diff
@@ -0,0 +1,104 @@
+//@import '~@automattic/marketplace/styles/mixins';
+
+.marketplace-domain-upsell__root {
+	margin-left: 34px;
+	display: grid;
+	grid-template-rows: 40px 100%;
+	grid-template-columns: 60% 40%;
+
+	*,
+	*::before,
+	*::after {
+		box-sizing: border-box;
+	}
+	
+	.marketplace-domain-upsell__shopping-cart-container {
+		padding-left: 50px;
+		max-width: 400px;
+	}
+
+	.marketplace-domain-upsell__header {
+		margin: 0 0 20px;
+		@media ( min-width: 782px ) {
+			margin: 0 0 20px;
+		}
+		@media ( min-width: 600px ) {
+			margin: 0 0 20px;
+		}
+	}
+	.marketplace-title {
+		font-family: var( --p2-font-inter );
+		letter-spacing: 0.4px;
+		line-height: 28px;
```

Comment:
Do we really need to overwrite all those variables? Would be great if we can inherit them and use what's already declared. I think that overlaps a bit with what Jarda mentioned a couple of lines above.

Reply:
I was trying to make the design pixel perfect. However I will try to adopt a more inherit based flow 👍

Reply:
I'd love to discuss this with the design team that worked on it, but my general recommendation would be to use the common interface, components and elements as much as possible, and avoid introducing specific UIs built from scratch, because they end up being hard to maintain in the long run. I've also talked about this in https://github.com/Automattic/wp-calypso/pull/52081#discussion_r622023826

Reply:
> I was trying to make the design pixel perfect.

It shouldn't be the implementation goal to make things pixel perfect. Calypso has a pre-existing design language (actually, several of them 🙃) with conventions for colors, fonts, spacings... often encoded in existing components. New UIs should reuse that as much as possible.

On this page, the domain picker is from Gutenboarding, and the "Your Cart" component is very similar to what we have in Checkout:

<img width="914" alt="Screenshot 2021-04-28 at 12 54 16" src="https://user-images.githubusercontent.com/664258/116397152-88117080-a826-11eb-9a06-e7f693f62df1.png">

What flow is the Yoast page part of? Creating a new site? Or is it an upsell for an existing site? Let's make the page fit in into its surrounding environment.

> I think that overlaps a bit with what Jarda mentioned a couple of lines above.

When first seeing the styles, my intuition was: "there's too much CSS here, the UI shouldn't have so many styles". But I didn't know how to express it as something actionable at the time.

Reply:
Hey @jsnajdr thanks for this feedback, after looking at http://calypso.localhost:3000/devdocs/design I understand what you and @tyxla mean. I will see how I can reuse more existing components to implement this page 😄 
This is is the design discussion if you are interested! p9Jlb4-2vT-p2
CC - @olaolusoga @JanaMW27

Reply:
> This is is the design discussion if you are interested!

Thank you for sharing the context! The discussion seems to all about designing the interactions and navigating the labyrinth of Simple vs Atomic vs different plans combinations and very little about specific icon shapes or colors. That makes me think you have a lot of freedom when implementing these details.